ansible/roles/test/tasks/everflow_testbed/apply_config/acl_rule_persistent.json.j2

```diff
@@ -76,7 +76,7 @@
                                 },
                                 "l2": {
                                     "config": {
-                                        "ethertype": "4660"
+                                        "ethertype": "2048"
```

Comment:
Why make this change?

Reply:
Hi @wangxin,

It is because 4660 is an invalid ethertype which translates to 1234. We therefore changed it to a valid IPv4 ethertype(2048)

Reply:
I think the original intention was to test ethertype `1234`. 

Searched through the code base, besides the ansible playbook based everflow_testbed test, the only pytest script using this template is the show_techsupport test. The show_techsupport test does not really care about the content details of the acl rule json. It was just to add/remove mirror session and check if expected log is found in syslog.

So, this change is unnecessary, and it also does not hurt. I am OK with it then.

Reply:
I agree with Xin. The change is unnecessary, but also no hurt.